kurl_util/cmd/bashmerge/main.go

```diff
@@ -45,6 +45,7 @@ func getInstallerConfigFromYaml(yamlPath string) (*kurlv1beta1.Installer, error)
 
 func checkIfFlagHasValue(length int, flag string) bool {
 	shouldHaveLengthTwo := []string{
+		"additional-no-proxy-addresses",
```

Comment:
Do we need this to be a bash flag? I suspect the answer is yes, but we're creeping back up in flags again.

Reply:
Yes, for the join script. I added a note to the design doc.